### PR TITLE
Justifying paragraphs on ScholarX 2023

### LINKED ISSUE #1618: Justifying the paragraphs in ScholarX 2023
### Is your feature request related to a problem? Please describe.
On the [ScholarX](https://sefglobal.org/scholarx/2023/) web page, only the paragraph related to the topic "Take part as a mentee under ScholarX23" has been justified, and the paragraphs related to other topics have not been justified.

### Describe the solution you'd like
Justifying the two paragraphs related to the remaining two topics

### Additional context
**Before Fixing the Issue**
before1
before2

**After Fixing the Issue**
after1
after2

### Are you working on it?
Yes

### PR DESCRIPTION
**Pull request by RamithaHeshan33**
## Purpose

The purpose of this PR is to fix #1618 

## Goals
Justifying the two paragraphs related to the remaining two topics in the [ScholarX](https://sefglobal.org/scholarx/2023/) web page

## Approach
approach

### Screenshots
**Before Fixing the Issue**

before1
before2

**After Fixing the Issue**

after1
after2
  
### Preview Link

https://pr-1619-sef-site.surge.sh/scholarx

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/2023/index.html [hunk 1]
```diff
@@ -167,11 +167,11 @@ <h1 class="display-3 text-center mb-5">Program Timeline</h1>
           <div class="row justify-content-center justify-content-md-end">
             <div class="col-9">
               <h3 class="mb-2 mt-4 text-onelive">Be a mentor for ScholarX23</h3>
-              <p>
+              <p class="text-justify">
                 Do you consider yourself a qualified professional with a knack for mentoring? 
                 Now you can share your knowledge with budding young talent of Sri Lanka.
-                Reach us via <a
-                href="mailto:sustainableedufoundation@gmail.com">sustainableedufoundation@gmail.com</a>
+                Reach us via <br>
```
Comment (YoshithaRathnayake) [inline scholarx/2023/index.html:173]: Can you explain why did you add a line brake here? @RamithaHeshan33

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: scholarx/2023/index.html (hunk 1 above)
Comment (RamithaHeshan33) [reply scholarx/2023/index.html:173]: Without a line break, the result is like this.

Screenshot 2024-04-25 222140

That's why I used a line break here @YoshithaRathnayake

Comment (YoshithaRathnayake) [reply scholarx/2023/index.html:173]: Ah ok @RamithaHeshan33

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Can you update the preview link? @RamithaHeshan33 
You should include the pr number except this {PR_NUMBER} and include the directory also
Ex: https://pr-0000-sef-site.surge.sh/scholarx

**Comment by RamithaHeshan33** [comment]
> Can you update the preview link? @RamithaHeshan33 You should include the pr number except this {PR_NUMBER} and include the directory also Ex: https://pr-0000-sef-site.surge.sh/scholarx

Sorry, I forgot that... Now updated!! @YoshithaRathnayake

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job!
Thank you @RamithaHeshan33
